docs/DeveloperGuide.md

```diff
@@ -157,6 +157,37 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Sessions
+The Session component consists of the following set of features: Create Session
+
+#### The Session class
+The Session Class encompasses several important attributes:
+* `SessionNumber`: This unique identifier helps distinguish one session from another. It is an integral part of the Session class and is a primary key when searching for or referencing sessions within the system.
+
+* `SessionStudents`: An essential component of every session is the list of students participating. The SessionStudents class keeps track of the students present in a particular session. This class allows for efficient management of attendance records and plays a vital role in generating attendance reports.
+
+* `SessionRemark`: Sometimes, additional information about a session is necessary, such as special instructions, topics covered, or any other relevant remarks. The SessionRemark field offers the flexibility to include such notes.
+
+The Session class also provides a set of getter methods that enable access to these attributes. For instance, you can retrieve the session number, list of students, or session remarks using these methods.
+
+Below is a class diagram describing the implementation of `Session` and its respective fields.
+
+###### NEED TO ADD CLASS DIAGRAM
```

Comment:
Probably to exclude placeholder as the UML Diagram is already present

Reply:
Nice catch! Owe you one :)